github/util_v4_repository.go

```diff
@@ -16,6 +16,12 @@ func getRepositoryID(name string, meta interface{}) (githubv4.ID, error) {
 		return githubv4.ID(name), nil
 	}
 
+	// Interpret `name` as a legacy node ID
+	exists, _ = repositoryLegacyNodeIDExists(name, meta)
```

Comment:
What's the reason the error is ignored here? This function already returns an error, so perhaps it's best to add a condition like the following:

```suggestion
	exists, err = repositoryLegacyNodeIDExists(name, meta)
	if err != nil {
	    return nil, err
	}
```

Reply:
Ohhh, I see, it's to continue to check the non-legacy case. Please disregard me! This looks good.